modules/serverless-tech-preview-features.adoc

```diff
@@ -6,23 +6,23 @@
 [id="serverless-tech-preview-features_{context}"]
 = Generally Available and Technology Preview features
 
-Features which are Generally Available (GA) are fully supported and are suitable for production use. Technology Preview (TP) features are experimental features and are not intended for production use. See the link:https://access.redhat.com/support/offerings/techpreview[Technology Preview scope of support on the Red Hat Customer Portal] for more information about TP features.
+Features that are Generally Available (GA) are fully supported and are suitable for production use. Technology Preview (TP) features are experimental features and are not intended for production use. See the link:https://access.redhat.com/support/offerings/techpreview[Technology Preview scope of support on the Red Hat Customer Portal] for more information about TP features.
 
 The following table provides information about which {ServerlessProductName} features are GA and which are TP:
 
 .Generally Available and Technology Preview features tracker
 [cols="2,1,1,1",options="header"]
 |====
-|Feature |1.28|1.29|1.30
+|Feature |1.29|1.30|1.31
 
 |Overriding `liveness` and `readiness` in functions
 |-
-|-
+|GA
 |GA
 
 |Eventing integration with {SMProductName}
 |-
-|-
+|TP
```

Comment:
We're now mentioning that:
* Eventing integration with Red Hat OpenShift Service Mesh - TP
* Service Mesh mTLS - GA

Where does the integration with Service Mesh being TP come from? I know we have one other part - Multi-tenancy with Service Mesh which is currently TP.

Reply:
* Using Service Mesh to isolate network traffic with OpenShift Serverless - it was not available at all in 1.29, 1.30. And it is TP in 1.31 for the first time.
* Service Mesh mTLS
  https://issues.redhat.com/browse/SRVKE-1410 - Developer Preview in 1.29
  https://issues.redhat.com/browse/SRVKE-1473 - Technical Preview in 1.30
  https://issues.redhat.com/browse/SRVKE-1450 - GA in 1.31
* Eventing integration with Red Hat OpenShift Service Mesh - Not sure if we want to mention this at all. It's an umbrella for both the above issues so I'm not sure if we want to say it's GA or TP. https://issues.redhat.com/browse/SRVKE-1311 but